### PR TITLE
Create Make targets to release changes to pi.builds

### PR DESCRIPTION
**Pull request by tobioboye**
Refs THUGS-1859

### CODE REVIEW

**Comment by jeffsawatzky** [comment]
I thought Travis-ci deployed to pypi for us? @DeanWay didn't you make that happen or am I crazy?

File: scripts/bump-version.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+# Needs to be run from the root directory of the project
+pip install bumpversion
```
Comment (jeffsawatzky) [inline scripts/bump-version.sh:5]: we may want to consider using bump2version which is a maintained fork of bumpversion and supports annotated git tags.

File: Makefile [hunk 2]
```diff
@@ -28,5 +35,59 @@ bump: tester
...
-	rm -rf ./build ./sparkplug.egg-info ./dist
+	rm -fr build/ dist/ *.egg-info
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+
+install:
+	pip install dist/sparkplug-*.whl
+
+test:
```
Comment (drone115b) [inline Makefile:57]: I think 'make test' is redundant with the existing 'make tests'

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,10 @@
+.PHONY: shell build install test clean release release-python2
+
+PYTHON_DOCKER_REPO=python:latest
+PYTHON_2_DOCKER_REPO=python:2-stretch
```
Comment (AntonNguyen) [inline Makefile:4]: Should we be encouraging this sort of behaviour? 🤔

**Comment by tobioboye** [comment]
Thanks for all the comments folks, I'll revisit them and put up a PR shortly

**Comment by DeanWay** [comment]
@jeffsawatzky you're correct, our travisci config should create and push a new version to pypi whenever a new tag is pushed. This should be removed in favour of documenting that

**Pull request merged**